src/repl.rs

```diff
@@ -239,38 +239,49 @@ impl<F: LurkField> ReplState<F> {
     pub fn handle_load<P: AsRef<Path>>(
         &mut self,
         store: &mut Store<F>,
-        path: P,
+        file_path: P,
         package: &Package,
     ) -> Result<()> {
-        println!("Loading from {}.", path.as_ref().to_str().unwrap());
-        self.handle_file(store, path.as_ref(), package, true)
+        println!("Loading from {}.", file_path.as_ref().to_str().unwrap());
+        self.handle_file(store, file_path.as_ref(), package, true)
     }
 
     pub fn handle_run<P: AsRef<Path> + Copy>(
         &mut self,
         store: &mut Store<F>,
-        path: P,
+        file_path: P,
         package: &Package,
     ) -> Result<()> {
-        println!("Running from {}.", path.as_ref().to_str().unwrap());
-        self.handle_file(store, path, package, false)
+        println!("Running from {}.", file_path.as_ref().to_str().unwrap());
+        self.handle_file(store, file_path, package, false)
     }
 
     pub fn handle_file<P: AsRef<Path> + Copy>(
         &mut self,
         store: &mut Store<F>,
-        path: P,
+        file_path: P,
         package: &Package,
         update_env: bool,
     ) -> Result<()> {
-        let p = path;
-
-        let input = read_to_string(path)?;
-        println!("Read from {}: {}", path.as_ref().to_str().unwrap(), input);
+        let file_path = file_path;
+
+        let input = read_to_string(file_path)?;
+        println!(
+            "Read from {}: {}",
+            file_path.as_ref().to_str().unwrap(),
+            input
+        );
         let mut chars = input.chars().peekmore();
 
         while self
-            .handle_form(store, &mut chars, package, p, update_env)
+            .handle_form(
+                store,
+                &mut chars,
+                package,
+                // use this file's dir as pwd for further loading
+                file_path.as_ref().parent().unwrap(),
```

Comment:
This is the actual fix right here, calling `.parent()`. Everything else is just name changes and formatting.